src/foraging_gui/Foraging.py

```diff
@@ -2743,20 +2749,27 @@ def _Open(self,open_last = False,input_file = ''):
                                 Tag=0
                         if type(value)==np.ndarray:
                             Tag=0
+
+                        if Tag==0:
```

Comment:
Can you use a more descriptive variable name than "Tag"

Reply:
Updated!